test/serverTest.js

```diff
@@ -200,6 +235,124 @@ describe('Brave Sensor server', () => {
     })
   })
 
+  describe('POST request radar and door events with mock im21 door sensor', () => {
```

Comment:
I think this is the same name as above, can you change them to specify XeThru vs Innosent?